refactor(dynamic-sampling): rename uniform bias (#966)
getsentry/develop: src/docs/dynamic-sampling/architecture.mdx

The invalidation of the configuration can happen under **several circumstances**, by calling the [following function](https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/relay.py#L244-L244). Some examples of cases when the recomputation happens are when a new release is detected, when some project settings change, or when the Celery tasks for computing the variable sample rates are finished executing.

The generation of the rules that will be part of the project configuration recalculation is defined [here](https://github.com/getsentry/sentry/blob/master/src/sentry/dynamic_sampling/rules/base.py#L72-L72) and works by performing the following steps:

1. Fetching the list of active biases, since some of them can be enabled or disabled by the user in the Sentry UI;

> 2. Determining the base sample rate which will be used for the low volume projects bias.

Reviewer: ```suggestion
2. Determining the base sample rate specific for each project.
```

Reviewer: Since now we updated how the system works, we should be more generic about this. The base sample rate is already generated using the low volume projects algorithm.
